platformio.ini

```diff
@@ -15,8 +15,9 @@ build_flags =
     -D SRC_REV=527
 
 # Sensor definition
-   -D HONEYWELL        ; if Honeywell uncomment
+   ;-D HONEYWELL       ; if Honeywell OR Plantower uncomment
    ;-D PANASONIC       ; if Panasonic uncomment
+   -D SENSIRION       ; if Sensirion uncomment
```

Comment:
The official sensor is PANASONIC for now, please change it.

Reply:
ok